backend/compact-connect/stacks/persistent_stack/bulk_uploads_bucket.py

```diff
@@ -145,6 +146,8 @@ def _add_ingest_object_events(self, event_bus: EventBus):
             entry=os.path.join('lambdas', 'license-data'),
             index=os.path.join('handlers', 'bulk_upload.py'),
             handler='process_s3_event',
+            timeout=Duration.minutes(15),
```

Comment:
I assume this is to allow for larger bulk uploads? How did you choose this #?

Reply:
15 minutes is the maximum timeout allowed on a lambda. I just wanted to allow for as much time reading the file as we could.